packages/atomic/src/locales.json

```
@@ -77,6 +77,10 @@
     "en": "Search suggestions. Select one to search.",
     "fr": "Suggestions de recherche. Sélectionnez-en un pour rechercher."
   },
+  "tab-search": {
```

@btaillon would you approve this PR again with this new aria label? I left it as 'facette' in French, let me know if there is a better word!

---

@liviarett "facette" would be the French equivalent to "Facet". The French equivalent to "tab" would've been "onglet".

This is what I think the string should've been:
```suggestion
    "en": "{{label}} tab.",
    "fr": "Onglet {{label}}"
```

I'm going to add it in an unrelated PR.